superset-frontend/src/profile/components/UserInfo.tsx

```diff
@@ -42,8 +47,8 @@ export default function UserInfo({ user }: UserInfoProps) {
         />
       </a>
       <hr />
-      <Panel>
-        <Panel.Body>
+      <div className="panel">
+        <div className="header">
```

Comment:
Is this div even necessary? Doesn't look like it adds any styles.

Reply:
Yah this is needed.